latch_cli/snakemake/single_task_snakemake.py

```diff
@@ -154,10 +160,26 @@ def emit_overrides(self, token):
     else:
         raise ValueError(f"tried to emit overrides for unknown state: {type(self)}")
 
-    positional_data = (render_annotated_str_list(x) for x in xs["positional"])
-    keyword_data = (
-        f"{k}={render_annotated_str_list(v)}" for k, v in xs["keyword"].items()
-    )
+    if (
+        isinstance(self, Output)
+        and xs["positional"][0].get("flags")
+        and "multiext" in xs["positional"][0].get("flags")
+    ):
+        eprint("inside")
+        quote = "'"
+        positional_data = (
+            (
+                f"multiext('{xs[0]['value']}',{','.join([quote + x['value'].split('.')[-1] + quote for x in xs])})"
+            ),
+        )
```

Comment:
```suggestion
        positional_data = (
            (
                f"multiext({repr(xs[0]['value'])},{','.join([repr(x['value'].split('.')[-1]) for x in xs])})"
            ),
        )
```

Reply:
also shouldnt it be `xs["positional"]` not just `xs`

Reply:
yes